src/widget/wcuemenupopup.h

```diff
@@ -52,10 +72,17 @@ class WCueMenuPopup : public QWidget {
   private slots:
     void slotEditLabel();
     void slotDeleteCue();
+    void slotUpdate();
+    void slotSavedLoopCue();
+    void slotAdjustSavedLoopCue();
```

Comment:
Could we call the slots
```suggestion
    void slotSavedLoopCueAuto();
    void slotSavedLoopCueManual();
```
to better reflect the difference?

Reply:
Not sure this makes it more clear, to me this feels even more confusing. How about I add a docstring to make it clearer?

Reply:
For me it's just that `slotAdjustSavedLoopCue` sounds like we adjust an existing loop cue, but the only difference to `slotSavedLoopCue` is that is uses the position instead of an auto-length.

Reply:
I see. With the docstring, I feel like those slots behaviour are well defined to developer, so happy to rename the symbol to anything we think is better fit.

Reply:
Thank you!